<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoomisDockerForm</class>
    <widget class="QWidget" name="LoomisDockerForm">
        <property name="windowTitle">
            <string>Loomis Head Controls</string>
        </property>

        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="leftMargin">
                <number>10</number>
            </property>
            <property name="topMargin">
                <number>10</number>
            </property>
            <property name="rightMargin">
                <number>10</number>
            </property>
            <property name="bottomMargin">
                <number>10</number>
            </property>

            <!-- Trackball host -->
            <item>
                <widget class="QWidget" name="trackballHost" native="true">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                            <horstretch>0</horstretch>
                            <verstretch>2</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="minimumSize">
                        <size>
                            <width>400</width>
                            <height>400</height>
                        </size>
                    </property>
                </widget>
            </item>

            <!-- Separation line -->
            <item>
                <widget class="QFrame" name="separatorTop">
                    <property name="frameShape">
                        <enum>QFrame::HLine</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Sunken</enum>
                    </property>
                </widget>
            </item>

            <!-- Extra padding below the trackball -->
            <item>
                <spacer name="spacerBelowTrackball">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>8</height>
                        </size>
                    </property>
                </spacer>
            </item>

            <!-- Controls (sliders) -->
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <property name="fieldGrowthPolicy">
                        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                    </property>
                    <property name="labelAlignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <property name="rowWrapPolicy">
                        <enum>QFormLayout::DontWrapRows</enum>
                    </property>
                    <property name="labelAlignment">
                        <set>Qt::AlignRight|Qt::AlignVCenter</set>
                    </property>
                    <property name="formAlignment">
                        <set>Qt::AlignLeft|Qt::AlignTop</set>
                    </property>
                    <property name="horizontalSpacing">
                        <number>12</number>
                    </property>
                    <property name="verticalSpacing">
                        <number>8</number>
                    </property>

                    <!-- Scale -->
                    <item row="0" column="0">
                        <widget class="QLabel" name="labelScale">
                            <property name="text">
                                <string>&amp;Scale:</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignVCenter</set>
                            </property>
                            <property name="buddy">
                                <cstring>sizeSlider</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QSlider" name="sizeSlider">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>10</number>
                            </property>
                            <property name="maximum">
                                <number>200</number>
                            </property>
                            <property name="value">
                                <number>100</number>
                            </property>
                            <property name="singleStep">
                                <number>1</number>
                            </property>
                            <property name="pageStep">
                                <number>5</number>
                            </property>
                        </widget>
                    </item>

                    <!-- Side cut -->
                    <item row="1" column="0">
                        <widget class="QLabel" name="labelSideCut">
                            <property name="text">
                                <string>Side cut:</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignVCenter</set>
                            </property>
                            <property name="buddy">
                                <cstring>sideCutSlider</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QSlider" name="sideCutSlider">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>5</number>
                            </property>
                            <property name="maximum">
                                <number>95</number>
                            </property>
                            <property name="value">
                                <number>66</number>
                            </property>
                            <property name="singleStep">
                                <number>1</number>
                            </property>
                            <property name="pageStep">
                                <number>5</number>
                            </property>
                        </widget>
                    </item>

                    <!-- Front stroke -->
                    <item row="2" column="0">
                        <widget class="QLabel" name="labelFrontStroke">
                            <property name="text">
                                <string>Front stroke:</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignVCenter</set>
                            </property>
                            <property name="buddy">
                                <cstring>frontStrokeSlider</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QSlider" name="frontStrokeSlider">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="maximum">
                                <number>12</number>
                            </property>
                            <property name="value">
                                <number>6</number>
                            </property>
                            <property name="singleStep">
                                <number>1</number>
                            </property>
                            <property name="pageStep">
                                <number>1</number>
                            </property>
                        </widget>
                    </item>

                    <!-- Back stroke -->
                    <item row="3" column="0">
                        <widget class="QLabel" name="labelBackStroke">
                            <property name="text">
                                <string>Back stroke:</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignVCenter</set>
                            </property>
                            <property name="buddy">
                                <cstring>backStrokeSlider</cstring>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QSlider" name="backStrokeSlider">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="minimum">
                                <number>0</number>
                            </property>
                            <property name="maximum">
                                <number>12</number>
                            </property>
                            <property name="value">
                                <number>5</number>
                            </property>
                            <property name="singleStep">
                                <number>1</number>
                            </property>
                            <property name="pageStep">
                                <number>1</number>
                            </property>
                        </widget>
                    </item>
                    <!-- Stroke color -->
                    <item row="4" column="0">
                        <widget class="QLabel" name="labelStrokeColor">
                            <property name="text">
                                <string>Stroke color:</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="1">
                        <layout class="QHBoxLayout" name="strokeColorLayout">
                            <property name="spacing">
                                <number>6</number>
                            </property>
                            <item>
                                <widget class="QFrame" name="strokeColorSwatch">
                                    <property name="minimumSize">
                                        <size>
                                            <width>28</width>
                                            <height>18</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: #6A54E7; border: 1px
                                            solid #666;</string>
                                    </property>
                                    <property name="frameShape">
                                        <enum>QFrame::Panel</enum>
                                    </property>
                                    <property name="frameShadow">
                                        <enum>QFrame::Sunken</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="strokeColorButton">
                                    <property name="text">
                                        <string>Choose…</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="strokeColorStretch">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Expanding</enum>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>

                </layout>

            </item>

            <!-- Guides (checkboxes grouped below sliders) -->
            <item>
                <widget class="QGroupBox" name="groupGuides">
                    <property name="title">
                        <string>Guides</string>
                    </property>
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <layout class="QVBoxLayout" name="guidesLayout">
                        <property name="leftMargin">
                            <number>8</number>
                        </property>
                        <property name="topMargin">
                            <number>6</number>
                        </property>
                        <property name="rightMargin">
                            <number>8</number>
                        </property>
                        <property name="bottomMargin">
                            <number>6</number>
                        </property>
                        <property name="spacing">
                            <number>4</number>
                        </property>
                        <item>
                            <widget class="QCheckBox" name="showSilhouette">
                                <property name="text">
                                    <string>Show Silhouette</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="showSideRims">
                                <property name="text">
                                    <string>Show Side Rims</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="showSideCross">
                                <property name="text">
                                    <string>Show Side Cross</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="showArrow">
                                <property name="text">
                                    <string>Show Arrow</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>

            <!-- Actions -->
            <item>
                <layout class="QHBoxLayout" name="rowActions">
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <layout class="QHBoxLayout" name="actions">
                            <property name="spacing">
                                <number>6</number>
                            </property>
                            <item>
                                <widget class="QPushButton" name="resetButton">
                                    <property name="text">
                                        <string>Reset Rotation</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="saveButton">
                                    <property name="text">
                                        <string>Save and close window</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>

        </layout>
    </widget>
    <resources />
    <connections />
</ui>